src/sentry/stacktraces/processing.py

```diff
@@ -220,13 +222,22 @@ def _append_stacktrace(stacktrace: Any, container: Any, is_exception: bool = Fal
         _append_stacktrace(thread.get("stacktrace"), thread)
 
     if include_raw:
-        for info in rv[:]:
-            if info.container is not None:
```

Comment:
This is another spot where https://github.com/getsentry/sentry/pull/15795#discussion_r349916876 might come into play. IDK if that use case requires empty dictionaries to be included.